src/research.py

```diff
@@ -88,89 +115,311 @@ def get_info(self):
+throwing_axes = Weapon('Throwing Axes', 60, 100, 0, 10, 0, 1.8, 1, reload_time=70, ammunition=3, projectile_speed=5, armor_pierce=-1, projectile_size=3)
+throwing_daggers = Weapon('Throwing Daggers', 20, 90, 0, 4, 0, 1.8, 1, reload_time=50, ammunition=5, projectile_speed=6, armor_pierce=-1, projectile_size=2)
+jarid = Weapon('Jarids', 60, 150, 0, 4, 0, 1.8, 1, reload_time=60, ammunition=10, projectile_speed=6, armor_pierce=1, projectile_size=10)
+stones = Weapon('Stones', 0, 80, 0, 1, 0, 1.8, 1, reload_time=30, ammunition=100, projectile_speed=3, armor_pierce=-1, projectile_size=1)
+darts = Weapon('Darts', 3, 80, 0, 2, 0, 1.8, 1, reload_time=20, ammunition=30, projectile_speed=5, armor_pierce=1, projectile_size=1)
+pilum = Weapon('Pilum', 100, 200, 0, 12, 0, 2, 1, reload_time=10, ammunition=3, projectile_speed=6, armor_pierce=1, projectile_size=12)
+
+recurve_bow = Weapon('Recurve Bow', 200, 320, 0.8, 7, 1, 3, 1, reload_time=75, ammunition=15, projectile_speed=14, armor_pierce=1, projectile_size=7)
+war_bow = Weapon('War Bow', 185, 350, 0.5, 8, 0.5, 2, 0.5, reload_time=80, ammunition=15, projectile_speed=13, armor_pierce=1, projectile_size=8)
+self_bow = Weapon('Self Bow', 145, 300, 0.5, 6, 1, 2, 1, reload_time=70, ammunition=15, projectile_speed=12, armor_pierce=-1, projectile_size=6)
+hunting_bow = Weapon('Hunting Bow', 105, 280, 0.3, 4, 1, 2.5, 1, reload_time=60, ammunition=15, projectile_speed=10, armor_pierce=-1, projectile_size=4)
+
+arbalest = Weapon('Arbalest', 300, 500, 1.5, 12, 1, 1, 1, reload_time=150, ammunition=15, projectile_speed=30, armor_pierce=1, projectile_size=4)
+hunting_crossbow = Weapon('Hunting Crossbow', 200, 400, 1.5, 8, 1, 1, 1, reload_time=100, ammunition=15, projectile_speed=15, armor_pierce=-1, projectile_size=2)
+
+handgun = Weapon('Handgun', 500, 350, 1.5, 30, 1, 1, 1, reload_time=200, ammunition=15, projectile_speed=40, armor_pierce=1, projectile_size=1)
+flintlock_pistol = Weapon('Flintlock Pistol', 400, 200, 1.5, 20, 1, 1, 1, reload_time=170, ammunition=15, projectile_speed=30, projectile_size=1)
+flintlock_rifle = Weapon('Flintlock Rifle', 600, 380, 1.5, 33, 1, 1, 1, reload_time=250, ammunition=15, projectile_speed=50, armor_pierce=1, projectile_size=1)
+matchlock_rifle = Weapon('Matchlock Rifle', 650, 400, 1.5, 35, 1, 1, 1, reload_time=270, ammunition=15, projectile_speed=50, armor_pierce=1, projectile_size=1)
+blunderbuss = Weapon('Blunderbuss', 550, 200, 1.5, 50, 1, 1, 1, reload_time=300, ammunition=15, projectile_speed=30, projectile_size=2)
+arquebus = Weapon('Arquebus', 700, 400, 1.5, 40, 1, 1, 1, reload_time=300, ammunition=15, projectile_speed=45, armor_pierce=1, projectile_size=1)
+musket = Weapon('Musket', 1000, 500, 1.5, 50, 1, 1, 1, reload_time=320, ammunition=15, projectile_speed=60, armor_pierce=1, projectile_size=1)
+
+
+#Ranged Siege - Kenny Additions
```

Comment:
Eventually (not for this pull request), should probably add a new weapon list for this rather than sticking it in with everything else.